crates/gem_hypercore/src/models/user.rs

```diff
@@ -24,3 +24,16 @@ pub struct HypercoreUserFee {
     #[serde(deserialize_with = "deserialize_f64_from_str")]
     pub active_referral_discount: f64,
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct HypercoreLedgerUpdate {
+    pub hash: String,
+    pub delta: HypercoreLedgerDelta,
+}
+
+#[derive(Debug, Clone, Default, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct HypercoreLedgerDelta {
```

Comment:
```suggestion
pub struct LedgerDelta {
```

let's drop the prefix for all models?